src/content/protocols/pancakeswap-v2-bsc.md

```diff
@@ -0,0 +1,150 @@
+
+- **Fee extraction** – The multisig can redirect the 0.08% protocol fee of any pool to any address.
+
+- **Emissions steering** – By adding pools or changing allocation points in MasterChefV2, the multisig can starve existing LPs of CAKE or allocate outsized rewards to insider pools. This is equivalent to 40 CAKE a block limit.
+
+The 3-of-6 scheme offers redundancy in the event of a single key loss, but it centralizes control within a small group, whose interests may not align with those of the broader community. While the optional timelock feature helps reduce the risk of sudden parameter changes, it only addresses part of the MasterChef and CAKE yield mechanism, not the AMM protocol fee distribution. If either **MasterChefV2** or **MasterChefV1** were compromised, a malicious actor could redirect all rewards to their own pool, preventing CAKE from being burned as intended, and potentially depleting user pools of CAKE.
+
+> Upgradeability score: **Medium**
+
+## Autonomy
+
+PancakeSwap v2’s core AMM (Factory + Pair contracts) has **no external price oracle, keeper, or off-chain dependency**: swaps resolve only against on-chain reserves.  
+The CAKE **emission layer (MasterChef V1 + V2)**, however, introduces an internal dependency: LP rewards rely on a single token (CAKE) that is minted under the control of a 3-of-6 multisig.
+
+> Autonomy score: **Medium**
+
+## Exit Window
+
+- Most farm changes pass a Timelock (≥ 6 h delay) → users can exit before execution.
+- Protocol-fee switch in each Pair is immutable.
+- Admins cannot freeze LP withdrawals.
+
+> Exit-Window score: **High**
+
+## Accessibility
+
+| Access route  | Details                                                                                      | Maintainer   |
+| ------------- | -------------------------------------------------------------------------------------------- | ------------ |
+| **Web UI**    | <https://pancakeswap.finance>                                                                | Pancake Labs |
+| **Self-host** | MIT-licensed repo <https://github.com/pancakeswap/pancake-frontend> – build & serve anywhere | Community    |
+
+> Accessibility score: **Medium**
+
+# Technical Analysis
+
+## Contracts
+
+| Contract               | Address                                                                                                                |
+| ---------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| PancakeFactory         | [`0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73`](https://bscscan.com/address/0xcA143Ce32Fe78f1f7019d7d551a6402fC5350c73) |
+| PancakeRouter V2       | [`0x10ED43C718714eb63d5aA57B78B54704E256024E`](https://bscscan.com/address/0x10ED43C718714eb63d5aA57B78B54704E256024E) |
+| GnosisSafe #1          | [`0xcEba60280fb0ecd9A5A26A1552B90944770a4a0e`](https://bscscan.com/address/0xcEba60280fb0ecd9A5A26A1552B90944770a4a0e) |
```

Comment:
We list multisigs in the "Permission Owners" and the "Security Council" tables only